File: extract.js
const fs = require('fs');
const content = fs.readFileSync('landing_page_simplified_layout/code.html', 'utf8');
const match = content.match(/data:image\/png;base64,([^"']+)/);
if (match) {
    const b64 = match[1];
    fs.writeFileSync('app/logo.png', Buffer.from(b64, 'base64'));
    console.log("Success");
} else {
    console.log("Not found");
}

File: extract.py
import re, base64

with open('landing_page_simplified_layout/code.html', 'r', encoding='utf-8') as f:
    content = f.read()

match = re.search(r'data:image/png;base64,([^"\'\>]+)', content)
if match:
    image_data = base64.b64decode(match.group(1))
    with open('app/logo.png', 'wb') as f:
        f.write(image_data)
    print("Success")
else:
    print("Match not found")
